ctaplot/gammaboard/gammaboard.py

```diff
@@ -272,7 +272,8 @@ def plot_angular_resolution_reco(self, ax=None):
                                                                            ax=ax,
                                                                            label=self.name + '_reco',
                                                                            color=self.color,
-                                                                           **post_classification_opt,
+                                                                           # linewidth=2.0,
+                                                                           # **post_classification_opt,
```

Comment:
I think  linewidth should be removed and `**post_classification_opt`  uncommented, no ?

Reply:
we don't need `**post_classification_opt` anymore because we have now buttons to control the display of pre and post classification curves.
I remove the 2 commented lines